backend/inspirehep/records/serializers/json/jobs.py

```diff
@@ -17,9 +17,14 @@
     wrap_schema_class_with_metadata(JobsPublicSchema), index_name="records-jobs"
 )
 
+jobs_json_2 = JSONSerializer(
```

Comment:
if we add everything on `JobsPublicSchema` don't need to change anything here :)